Cargo.toml

```diff
@@ -20,7 +20,8 @@ exclude = [
   "resources/persist",
   "resources/secrets",
   "resources/shared-db",
-  "resources/static-folder"
+  "resources/static-folder",
+  "integrations"
```

Comment:
I'm not sure if we want to call these `services` instead?

Reply:
That makes sense, I'll rename it. :+1: